teuthology/task/install.py

```diff
@@ -790,6 +790,39 @@ def rh_install_pkgs(ctx, remote, version, rh_ds_yaml):
         raise RuntimeError("Version check failed on node %s", remote.shortname)
 
 
+def rh_install_deb_pkgs(
+        ctx,
+        remote,
+        version,
+        rh_ds_yaml,
+        deb_repo,
+        deb_gpg_key):
+    """
+    Setup debian repo, Install gpg key
+    and Install on debian packages
+    : param ctx
+    : param remote
```

Comment:
Please document all the args.

Also, same issue with ``rh_ds_yaml`` not being yaml, but a dict